From: Simon McVittie <smcv@debian.org>
Date: Thu, 27 Aug 2020 15:22:49 +0100
Subject: Skip a test for backward-compatible time zone aliases

This test depends on the precise behaviour of backward-compatible time
zone aliases, and fails with both the system ICU and the bundled copy.

Forwarded: no
---
 js/src/tests/jstests.list | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/js/src/tests/jstests.list b/js/src/tests/jstests.list
index 84fe75e..3ac423e 100644
--- a/js/src/tests/jstests.list
+++ b/js/src/tests/jstests.list
@@ -676,6 +676,9 @@ fails-if(xulRuntime.debian.DEB_HOST_ARCH.match(/^armel$/)) script test262/langua
 # Assumes little-endian IEEE representation of floating point numbers
 fails-if(xulRuntime.debian.DEB_HOST_ARCH_ENDIAN=="big") script test262/built-ins/TypedArray/prototype/set/typedarray-arg-set-values-same-buffer-other-type.js
 
+# Fails with both vendored and system ICU
+skip script non262/Intl/DateTimeFormat/timeZone_backward_links.js
+
 ###########################################################
 # Tests disabled due to issues in test262 importer script #
 ###########################################################
